pages/osx/dd.md

```diff
@@ -1,19 +0,0 @@
-# dd
```

Comment:
That difference is justified: I've tested `status=progress` on OSX and it results in "dd: unknown operand status". We should either remove `status=progress` from the common page, or keep the separate osx page.